README.md

```diff
@@ -16,13 +15,63 @@ And then execute:
 $ bundle
 ```
 
-Or install it yourself as:
-```bash
-$ gem install govuk_publishing_components
+### Mount the component guide
+
+Mount the component guide in your application:
+
+```ruby
+# config/routes.rb
+unless Rails.env.production?
+  mount GovukPublishingComponents::Engine, at: "/component-guide"
+end
+```
+
+For example, if your application was government-frontend, the component guide would be found at:
+`government-frontend.dev.gov.uk/component-guide`
+
+### Write components
+
+`govuk_publishing_components` expects the following components structure in your application:
+
+* a Partial View in `app/views/components` eg `_component-name.html.erb` - The template logic and markup, also defines the arguments expected
```

Comment:
Having an example file name here is helpful - can example file names also be added for the following items as well?